src/sequelize/migrations/20200314094556-create-roles-table.js

```diff
@@ -0,0 +1,41 @@
+module.exports.up = (queryInterface, DataTypes) => queryInterface.createTable('Roles', {
+  id: {
+    allowNull: false,
+    autoIncrement: true,
+    primaryKey: true,
+    type: DataTypes.INTEGER.UNSIGNED,
+  },
+  name: {
+    allowNull: false,
+    type: DataTypes.STRING,
+  },
+  description: {
+    allowNull: false,
+    type: DataTypes.TEXT,
+  },
+  userId: {
```

Comment:
I think we do not need the userId as we already referenced the roles using roleId

Reply:
worked on the feedback!